### ISSUE TITLE
Create base level manager

### ISSUE THREAD

**Issue by ddmills** [ddmills/big-waterbird#23]
The level manager component should reliably recreate a level given a seed.

Base level manager does not need to actually generate the levels yet, just choose between the available rooms.

**Comment by selfsame** [comment]
setting UnityEngine.Random.seed and then using using UnityEngine.Random.value will do this, we should have some static functions for randomNth on an array and stuff to make use of that

**Comment by ddmills** [comment]
agreed, and having an input field for the seed from the menu would be nice too
